cmd/telemetrygen/internal/traces/traces.go

```diff
@@ -123,13 +124,18 @@ func Run(c *Config, logger *zap.Logger) error {
 	}
 
 	var statusCode codes.Code
-	if c.StatusCode == "" {
+
+	switch strings.ToLower(c.StatusCode) {
+	case "0", "unset", "":
```

Comment:
I was going to use `codes.Unset.String()` here but went for hard coding the values since they are unlikely to change and its easier to read when having the mixed case functionality.